### PR TITLE
Update Definition of Done Team <team name>.md

### PR DESCRIPTION
**Pull request by barrystokman**
Definition of Definition of Done:

The team agrees on, and displays prominently somewhere in the team room, a list of criteria which must be met before a product increment "often a user story" is considered "done". Failure to meet these criteria at the end of a sprint normally implies that the work should not be counted toward that sprint's velocity.

https://www.agilealliance.org/glossary/definition-of-done

Version 0.1, please check if there's anything missing, unclear or spelled wrong.

### CODE REVIEW

File: scrum/Definition of Done Team <team name>.md [hunk 1]
```diff
@@ -1 +1,39 @@
...
+
+Scope: 
+* adding functionality
+* restructuring
+
+1) see *defect fix/patch*, additionally:  
+2) acceptance criteria/requirements should be met  
+3) automated unit tests for the new functionality created and passed  
+4) integration tests (component or system) should be passed  
+5) the change(s) in functionality should be documented in Github or AM
```
Comment (ingkebil) [inline scrum/Definition of Done Team <team name>.md]: Specify where in GH:
- if it changes *how* we develop: update the development repo.
- document the availablity of the new functionality (CLI) in the tool's docs
- update the method/procedure document in atlas

File: scrum/Definition of Done Team <team name>.md [hunk 2]
```diff
@@ -1 +1,39 @@
...
+## Definition of Done Team &lt;team name&gt;
+
+### 1) defect fix/patch
+
+Scope: 
+* fixing something that used to work before, but not anymore. There are no changes to functionality.
+
+1) all unit tests passed  
+2) pull request reviewed and approved  
+3) validation test (acceptance test) by an end user passed. There's no need for a test specification.
```
Comment (ingkebil) [inline scrum/Definition of Done Team <team name>.md]: Need to define how an acceptance is proposed.

File: scrum/Definition of Done Team <team name>.md (hunk 1 above)
Comment (ingkebil) [reply scrum/Definition of Done Team <team name>.md]: The actual decision tree outlined above, could be in a break out document. I like how each item concise and adding too many details might take away from that.

**Comment by ingkebil** [comment]
Also, this document has a lot of overlap with https://github.com/Clinical-Genomics/development/tree/master/validation#4-validation

As the definition of done outlines script-wise how to proceed, we can remove the #4-validation section from the validation document.

What do you think?

**Comment by patrikgrenfeldt** [comment]
@barrystokman what should we do with this PR?

**Comment by barrystokman** [comment]
We should close it, it's outdated.

**Comment by patrikgrenfeldt** [comment]
Closing, as suggested by barrystokman

**Pull request closed without merging**